CsvParser/csvParser.go

```diff
@@ -94,47 +98,158 @@ func CsvParser(filePath string, numIncidents ...int) []incident.DefaultIncident
 	return defaultIncidents
 }
 
-//* This function takes in an array of 100 default incidents because that is what is provided by the csvParser above *//
-func convertDefaultIncidentToIncident(defaultIncidents []incident.DefaultIncident) []incident.Incident {
-	incidentArr := make([]incident.Incident, len(defaultIncidents), len(defaultIncidents))
-	for i := range defaultIncidents {
-		incidentArr[i] = &defaultIncidents[i]
-	}
-	return incidentArr
+//convert a default incident that implements our interface to an incident object designed to exist in a json file
```

Comment:
Make comments consistently start with a space and an uppercase letter and change tone to remove "our", "us", "we"